server/public/pluginapi/group.go

```diff
@@ -167,3 +167,19 @@ func (g *GroupService) GetGroups(page, perPage int, opts model.GroupSearchOpts,
 	groups, appErr := g.api.GetGroups(page, perPage, opts, viewRestrictions)
 	return groups, normalizeAppErr(appErr)
 }
+
+// CreateDefaultMemberships creates default channel/team memberships for a group.
```

Comment:
Should we just copy the docs from `server/public/plugin/api.go`?

Reply:
Done